src/pages/FindGosu/FindGosu.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+// import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import GosuList from './GosuList';
+
+const FindGosu = () => {
+  const [gosuLists, setGosuLists] = useState([]);
+  const [selectedGosu, setSelectedGosu] = useState([]);
+  const [serviceId, setServiceId] = useState([]);
+  // const history = useHistory();
+
+  useEffect(() => {
+    fetch(`http://10.58.2.233:8000/applications/services/17/masters?user_id=15`)
```

Comment:
```suggestion
    fetch(`${API}/applications/services/17/masters?user_id=15`)
```
ip, 포트 번호 등은 src/config.js에 분리해주세요

Reply:
수정 완료!!!